planner/core/logical_plan_builder.go

```diff
@@ -1910,14 +1910,18 @@ func (b *PlanBuilder) checkOnlyFullGroupByWithGroupClause(p LogicalPlan, sel *as
 	gbyColNames := make(map[*types.FieldName]struct{}, len(sel.Fields.Fields))
 	gbyExprs := make([]ast.ExprNode, 0, len(sel.Fields.Fields))
 	for _, byItem := range sel.GroupBy.Items {
-		if colExpr, ok := byItem.Expr.(*ast.ColumnNameExpr); ok {
+		expr := byItem.Expr
+		if pe, ok := expr.(*ast.ParenthesesExpr); ok {
```

Comment:
Should the `UnaryOperationExpr` be handled? If so, just call the function directly?